lib/committee.js

```diff
@@ -3,18 +3,18 @@ import createDebug from 'debug'
 import { getTaskId } from './retrieval-stats.js'
 
 /** @import {Measurement} from './preprocess.js' */
-/** @import {RetrievalResult, CommitteeCheckError} from './typings.js' */
+/** @import {RetrievalResult, ConsensusCheckError} from './typings.js' */
 
 const debug = createDebug('spark:committee')
 
 /** @typedef {Map<string, Committee>} TaskIdToCommitteeMap */
 
 /** @typedef {{
    hasIndexMajority: boolean;
-   indexerResult: string  | CommitteeCheckError;
+   indexerResult: string | ConsensusCheckError;
```

Comment:
should this be called `ConsensusEvaluationError`? Otherwise we should call the other one `ConsensusCheckDetails`, to be consistent